lib/ansible/plugins/action/__init__.py

```diff
@@ -329,8 +329,8 @@ def _configure_module(self, module_name, module_args, task_vars) -> tuple[_Built
                     environment=final_environment,
                     remote_is_local=bool(getattr(self._connection, '_remote_is_local', False)),
                     become_plugin=self._connection.become,
+                    module_temp_dir=module_temp_dir,
```

Comment:
Looks like there's no such var in this context. See if you can get this value from somewhere and check for `None` too.